contracts/proxy-lib/tests/common/proxy_lib_helper.rs

```diff
@@ -4,26 +4,26 @@ use calimero_context_config::{
     Proposal, ProposalAction, ProposalApprovalWithSigner, ProposalId, ProxyMutateRequest,
 };
 use ed25519_dalek::{Signer, SigningKey};
-use near_workspaces::network::Sandbox;
+use near_sdk::AccountId;
 use near_workspaces::result::{ExecutionFinalResult, ViewResultDetails};
-use near_workspaces::{Account, Contract, Worker};
+use near_workspaces::{Account, Contract};
 use serde_json::json;
 
-use super::deploy_contract;
-
-const PROXY_CONTRACT_WASM: &str = "./res/proxy_lib.wasm";
+pub const PROXY_CONTRACT_WASM: &str = "./res/proxy_lib.wasm";
 
 pub struct ProxyContractHelper {
-    pub proxy_contract: Contract,
-    config_contract: Contract,
+    pub proxy_contract: AccountId,
+    config_contract: AccountId,
 }
 
 impl ProxyContractHelper {
-    pub async fn new(worker: &Worker<Sandbox>, config_contract: Contract) -> eyre::Result<Self> {
-        let proxy_contract = deploy_contract(worker, PROXY_CONTRACT_WASM).await?;
+    pub async fn new(
+        proxy_contract: &AccountId,
+        config_contract: &AccountId,
+    ) -> eyre::Result<Self> {
         Ok(Self {
-            proxy_contract,
-            config_contract,
+            proxy_contract: proxy_contract.clone(),
+            config_contract: config_contract.clone(),
```

Comment:
if you're going to clone, don't take references

```suggestion
        proxy_contract: AccountId,
        config_contract: AccountId,
    ) -> eyre::Result<Self> {
        Ok(Self {
            proxy_contract,
            config_contract,
```

let the caller determine how they want to derive and pass owned values